test/envoye2e/stats_plugin/stats_test.go

```diff
@@ -123,6 +123,20 @@ var TestCases = []struct {
 	},
 }
 
+var AttributeGenRuntimes = []struct {
+	AttributeGenFilterCode  string
+	WasmRuntime             string
+}{
+	{
+		AttributeGenFilterCode:    "inline_string: \"envoy.wasm.attributegen\"",
+		WasmRuntime:        	     "envoy.wasm.runtime.null",
```

Comment:
nit: seems like lots of places in the file that are not properly aligned. For example L132, L136, L324, etc. Could you use gofmt or the tools in your IDE to format the file?